terraform/gke/README.md

```diff
@@ -440,16 +440,26 @@ similar process.
     [Troubleshooting](#troubleshooting) section for more details.
 
     If you have chosen to use your own Spanner instance, please edit the above
-    configuration file accordingly.
+    configuration files accordingly.
 
 10.  To configure the Autoscaler and begin scaling operations, run the following
      command:
 
      ```sh
-     kubectl apply -f autoscaler-config/autoscaler-config.yaml
+     kubectl apply -f autoscaler-config/
      ```
 
-11.  Any changes made to the configuration file and applied with `kubectl
+     The sample configuration creates two schedules for scaling operations; a
```

Comment:
I think this note should go earlier. 
Suggestion: in step #6 explaining that is the reason why we have 2 yaml files under autoscaler-pkg (2 CronJobs) and 2 in autoscaler-config (2 ConfigMaps).